README.md

```diff
@@ -25,7 +25,7 @@ This package includes command line utilities that interacts with LiveKit. It all
 ## Mac
 
 ```shell
-brew install livekit-cli
+brew install lk
```

Comment:
Is it possible to bind short names like this on Brew?

Reply:
Nope! We aren't renaming the formula. Readme changes are stale, I'll fix this.